### PR TITLE
User namespaces.md 1

### PR DESCRIPTION
**Pull request by YashPimple**
Made Changes in line 93 
Issue #36867

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/user-namespaces.md [hunk 1]
```diff
@@ -89,10 +89,9 @@ isolate the users in the container from the users in the node.
 This means containers can run as root and be mapped to a non-root user on the
 host. Inside the container the process will think it is running as root (and
 therefore tools like `apt`, `yum`, etc. work fine), while in reality the process
-doesn't have privileges on the host. You can verify this, for example, if you
-check the user the container process is running `ps` from the host. The user
-`ps` shows is not the same as the user you see if you execute inside the
-container the command `id`.
+doesn't have privileges on the host. You can verify this, for example, if you check
+the user container the process is running `ps` from the host. The user output
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/pods/user-namespaces.md:93]: how do you parse "if you check the user container the process is running `ps` from the host"?

What is the objective for the verb "check"? "user container"? Then what is "the process is running `ps` from the host"? 

This sentence is not valid English syntax, right?

Comment (sftim) [reply content/en/docs/concepts/workloads/pods/user-namespaces.md:93]: I also don't understand this sentence.

**Comment by sftim** [comment]
@YashPimple please edit the title and description for this PR to summarize the change you are making.

We need that detail so we can check that the change you're making matches up with that summary.

**Comment by divya-mohan0209** [comment]
@YashPimple : Any update?

**Comment by YashPimple** [comment]
@divya-mohan0209 working on it and will raise a PR soon.

**Comment by divya-mohan0209** [comment]
@YashPimple - Please feel free to reopen this PR when you have some cycles to complete it :) Also, kindly rebase whenever you do so.
/close

**Pull request closed without merging**